File: app/src/main/java/de/tum/in/tumcampus/models/Identity.java
package de.tum.in.tumcampus.models;

import org.simpleframework.xml.Root;
import org.simpleframework.xml.Element;

@SuppressWarnings("UnusedDeclaration")
@Root(name = "row")
public class Identity {
    @Element
    private String vorname;

    @Element
    private String familienname;

    @Element
    public String kennung;

    @Override
    public String toString() {
        return vorname+" "+familienname;
    }
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/person/model/Identity.kt
package de.tum.`in`.tumcampusapp.component.tumui.person.model

import com.tickaroo.tikxml.annotation.Element
import com.tickaroo.tikxml.annotation.PropertyElement
import com.tickaroo.tikxml.annotation.Xml

@Xml(name = "row")
data class Identity(
    @PropertyElement(name = "vorname") val firstName: String = "",
    @PropertyElement(name = "familienname") val lastName: String = "",
    @PropertyElement(name = "kennung") val id: String = "",
    @PropertyElement val obfuscated_id: String = "",
    @Element val obfuscated_ids: ObfuscatedIds = ObfuscatedIds()
) {
    override fun toString(): String = "$firstName $lastName"
}
